src/appmixer/salt/auth.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+module.exports = {
+    type: 'apiKey',
+
+    definition: {
+        auth: {
+            apiKey: {
+                type: 'text',
+                name: 'API Key',
+                tooltip: 'Log into your Service account and find <i>API Keys</i> page.'
+            }
+        },
+
+        requestProfileInfo(context) {
+            const apiKey = context.apiKey;
```

Comment:
Add a newline after function declaration.
```suggestion
        requestProfileInfo(context) {

            const apiKey = context.apiKey;
```